### PR TITLE
Update starter app

### PR DESCRIPTION
**Pull request by paulcastro**
Update starter app to initially invoke utils/date instead of util/date to stop 403 error.

add privacy text description to Info.plist to squash warning about missing privacy text description in Info.plist.

### CODE REVIEW

File: mobile/iOS/starterapp/Podfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ use_frameworks!
 
 target 'OpenWhiskStarterApp' do
      platform :ios, '10.0'
-     pod 'OpenWhisk', :git => 'https://github.com/openwhisk/openwhisk-client-swift.git', :tag => '0.2.1'
+     pod 'OpenWhisk', :git => 'https://github.com/openwhisk/openwhisk-client-swift.git', :tag => '0.2.2'
```
Comment (csantanapr) [inline mobile/iOS/starterapp/Podfile:5]: 0.2.4 ?

Comment (paulcastro) [reply mobile/iOS/starterapp/Podfile:5]: I'd like to keep the SDK version at 0.2.2 since it hasn't changed.  Just the starter app.

Comment (csantanapr) [reply mobile/iOS/starterapp/Podfile:5]: Sounds good, I was just double checking

**Review by jberstler — APPROVED** [review]
LGTM

**Pull request merged**